src/nvim/drawline.c

```diff
@@ -106,6 +106,8 @@ typedef struct {
   int c_extra;               ///< extra chars, all the same
   int c_final;               ///< final char, mandatory if set
 
+  colnr_T n_closing;         ///< number of chars in fdc which will be closing
```

Comment:
I think this should just be an `int`
```suggestion
  int n_closing;             ///< number of chars in fdc which will be closing
```